package.json

```diff
@@ -58,9 +57,9 @@
     "screwdriver-request": "^1.0.1"
   },
   "release": {
-    "debug": false,
-    "verifyConditions": {
-      "path": "./node_modules/semantic-release/src/lib/plugin-noop.js"
-    }
+    "debug": false
+  },
+  "engines": {
+    "node": ">=14.0.0"
```

Comment:
should we stick to 12 till we update every where ? https://github.com/screwdriver-cd/screwdriver/blob/master/package.json#L27

Reply:
oh okay sure
I was preemptively updating to help with this issue (https://github.com/screwdriver-cd/screwdriver/issues/2683) but I can do node:12 for now